FilmsApp/View/GeneralController/MainViewController.swift

```diff
@@ -0,0 +1,102 @@
+import UIKit
+
+class ViewController: UIViewController {
+    
+    var myTableView = UITableView()
+    var film: FilmsCollection?
+    var presenter: MainViewPresenterProtocol?
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        createTableView()
+        self.navigationItem.title = "Films"
+        self.navigationController?.navigationBar.prefersLargeTitles = true
+        
+    }
+    
+    init(presenter: MainViewPresenterProtocol) {
+        self.presenter = presenter
+        super.init(nibName: nil, bundle: nil)
+        self.presenter?.view = self        
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    private func createTableView() {
+        myTableView = UITableView(frame: view.bounds, style: .plain)
+        self.view.addSubview(myTableView)
+        self.myTableView.register(TableViewCell.self, forCellReuseIdentifier: "cell")
+        self.myTableView.delegate = self
+        self.myTableView.dataSource = self
+        self.myTableView.backgroundColor = .white
+        myTableView.isUserInteractionEnabled = true
```

Comment:
This can be moved to an initialization block, the same as you were initializing collectionView